scrapy/extensions/feedexport.py

```diff
@@ -151,7 +151,7 @@ def __init__(self, settings):
         if not self._exporter_supported(self.format):
             raise NotConfigured
         self.store_empty = settings.getbool('FEED_STORE_EMPTY')
-        self.export_fields = settings.getlist('FEED_EXPORT_FIELDS')
+        self.export_fields = settings.getlist('FEED_EXPORT_FIELDS') or None
```

Comment:
Sorry, I meant that users won't be able to disable all fields to export, not that FEED_EXPORTS couldn't be set to `[]`.

I agree with those Settings changes requiring more discussion, and using the current Settings implementation (with its limitations) for now.

Maybe we could check for explicit `[]`s just to leave everyone happy?
```python
if settings.get('FEED_EXPORT_FIELDS') is not None:
     self.export_fields = settings.getlist('FEED_EXPORT_FIELDS')
else:
     self.export_fields = None
```